packages/grafana-ui/src/components/OptionsUI/color.tsx

```diff
@@ -24,7 +24,7 @@ export const ColorValueEditor: React.FC<ColorValueEditorProps> = ({ value, onCha
     <ColorPicker color={value ?? ''} onChange={onChange} enableNamedColors={true}>
       {({ ref, showColorPicker, hideColorPicker }) => {
         return (
-          <div className={styles.spot} onBlur={hideColorPicker}>
+          <div className={styles.spot}>
```

Comment:
Was the removal of the `onBlur` handler intentional?

Reply:
Ok great catch, in the future it would better to open a separate issue for this issue and handle it in a separate PR.